docs/guide/replacement-guides/qs.md

```diff
@@ -0,0 +1,82 @@
+---
+description: Modern alternatives to the qs package for parsing and serializing query strings
+---
+
+# Replacements for `qs`
+
+## `URLSearchParams`
+
+[`URLSearchParams`](https://developer.mozilla.org/docs/Web/API/URLSearchParams) is built into browsers and Node.js (>= 10). Use it when you don’t need nested objects or automatic array parsing. It preserves multiple values via `getAll`, and `toString()` gives you a URL-safe query string.
+
+Example:
+
+```ts
+import qs from 'qs' // [!code --]
+
+const query = 'a=1&a=2&b=3' // [!code ++]
+const obj = qs.parse(query) // { a: ['1','2'], b: '3' } [!code --]
+const sp = new URLSearchParams(query) // [!code ++]
+const obj = Object.fromEntries(sp) // { a: '2', b: '3' } [!code ++]
+const a = sp.getAll('a') // ['1', '2'] [!code ++]
+
+const input = { a: ['1', '2'], b: '3' } // [!code ++]
+const str = qs.stringify(input) // [!code --]
+const out = new URLSearchParams() // [!code ++]
+for (const v of input.a) out.append('a', v) // [!code ++]
+out.append('b', input.b) // [!code ++]
+const str = out.toString() // 'a=1&a=2&b=3' [!code ++]
+```
```

Comment:
this diff doesn't make much sense i think

most of it didn't exist before and only exists after. remember these diffs are meant to look like an actual, real diff you might encounter by switching